code/game/objects/items/weapons/stunbaton.dm

```diff
@@ -83,6 +83,21 @@
 /obj/item/melee/baton/get_cell()
 	return cell
 
+/obj/item/melee/baton/mob_can_equip(mob/user, slot, disable_warning = 1)
+	if(turned_on && slot == slot_belt)
+		to_chat(user, "<span class='warning'>You can't equip [src] while it's active!</span>")
+		return FALSE
+	if(turned_on && slot == slot_s_store)
+		to_chat(user, "<span class='warning'>You can't equip [src] while it's active!</span>")
+		return FALSE
+	return ..(user, slot, disable_warning == TRUE) // call parent but disable warning
```

Comment:
```suggestion
	return ..(user, slot, TRUE) // call parent but disable warning
```
No need to name the arg here

Reply:
Also, for future reference, you'd want to do `disable_warning = TRUE` instead of `disable_warning == TRUE`, the latter is a boolean value instead of naming an arg.

Reply:
I would prefer the args to be named as it makes the code more readible. but you need to remove the double `==`

Reply:
Done